packages/diff-graphics-magick/src/create-graphics-magick-differ.js

```diff
@@ -1,9 +1,21 @@
 const fs = require('fs-extra');
 const gm = require('gm');
+const imageSize = require('image-size');
 
 function createGraphicsMagickDiffer(config) {
   return function getImageDiff(path1, path2, diffPath, tolerance) {
     const instanceConfig = { tolerance: tolerance / 100, ...config };
+
+    const path1Dimensions = imageSize(path1);
```

Comment:
Can't we use [`gm.size`](https://aheckmann.github.io/gm/docs.html#getters) instead so we can lose the added dependency?

Reply:
replaced image-size with gm.size